### PR TITLE
W2 - vietphan - Create RecyclerView

### PR DESCRIPTION
**Pull request by at-vietphan**
#### Related issues (Optional)
Task #21 

#### Description (Required)
- [x] Create screen
- [x] Defines models class
- [x] Bind Data
- [x] Handle button and set color count like

### CODE REVIEW

File: app/src/at-vietphan/java/vn/asiantech/internship/RecyclerViewActivity.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        initData();
+        initAdapter();
+    }
+
+    private void initViews() {
+        mRecyclerViewStatus = findViewById(R.id.recyclerViewStatus);
+    }
+
+    private void initData() {
+        mStatusList.add(new Status("m_d", "Content Content Content Content", 1));
```
Comment (at-dev-support-01) [inline app/src/at-vietphan/java/vn/asiantech/internship/RecyclerViewActivity.java]: WARNING: [AvoidDuplicateLiterals] The String literal "Content Content Content Content" appears 6 times in this file; the first occurrence is on line 32
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals

**Comment by at-vietphan** [comment]
@at-huongnguyen Please review this pull request. Thanks :+1:

File: app/src/at-vietphan/java/vn/asiantech/internship/RecyclerViewActivity.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        initData();
+        initAdapter();
+    }
+
+    private void initViews() {
+        mRecyclerViewStatus = findViewById(R.id.recyclerViewStatus);
+    }
+
+    private void initData() {
+        mStatusList.add(new Status("m_d 1", "Content Content Content Content", 1));
```
Comment (at-dev-support-01) [inline app/src/at-vietphan/java/vn/asiantech/internship/RecyclerViewActivity.java]: WARNING: [AvoidDuplicateLiterals] The String literal "Content Content Content Content" appears 6 times in this file; the first occurrence is on line 32
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals

**Comment by at-vietphan** [comment]
@at-huongnguyen Please review this pull request. Thanks :+1:

File: app/src/at-vietphan/AndroidManifest.xml [hunk 3]
```diff
@@ -9,13 +9,13 @@
...
         android:theme="@style/AppTheme">
-        <activity android:name="vn.asiantech.internship.MainActivity">
+        <activity android:name=".MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".RecyclerViewActivity"></activity>
```
Comment (at-huongnguyen) [inline app/src/at-vietphan/AndroidManifest.xml]: Update to: activity android:name=".RecyclerViewActivity"/>

File: app/src/at-vietphan/res/layout/item_status.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    <TextView
+        android:id="@+id/tvContent"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:ellipsize="end"
+        android:maxLines="2"
+        android:paddingLeft="8dp"
+        android:paddingTop="60dp"
+        apps:text="@string/text_example_status_content" />
```
Comment (at-huongnguyen) [inline app/src/at-vietphan/res/layout/item_status.xml]: Should use tools:text

File: app/src/at-vietphan/res/layout/item_status.xml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+            android:layout_height="match_parent"
+            android:background="@drawable/selector_btn_dislike"
+            android:src="@drawable/ic_like_default" />
+
+        <TextView
+            android:id="@+id/tvCountLike"
+            android:layout_width="30dp"
+            android:layout_height="match_parent"
+            android:gravity="center"
+            apps:text="10" />
```
Comment (at-huongnguyen) [inline app/src/at-vietphan/res/layout/item_status.xml]: Should use tools:text

**Comment by at-vietphan** [comment]
@at-huongnguyen Please review this pull request. Thanks :+1:

**Comment by at-vietphan** [comment]
@at-huongnguyen Please review this pull request. Thanks :+1:

**Pull request merged**